test/expected/mongo/db-analysis-output/deep-nested-fields.json

```diff
@@ -0,0 +1,92 @@
+{
+  "persons": {
+    "fields": [
+      {
+        "name": "name",
+        "type": "String"
+      },
+      {
+        "name": "very",
+        "type": {
+          "deep": {
+            "model": {
```

Comment:
`deep` and `model` prop do not have the same model than their parent, `very`
`very' is the value of 'name' prop, but `deep` and `model` are directly used as keys.
Is this first level vs deep levels?
At first sight, I think we should have the same structure independently from the level.

Reply:
I don't get you there sorry 😥 Ping me to discuss this together